sdk/workloads/azure-resourcemanager-workloads/src/test/java/com/azure/resourcemanager/workloads/WorkloadsManagerTests.java

```diff
@@ -0,0 +1,153 @@
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.DoNotRecord;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.loganalytics.LogAnalyticsManager;
+import com.azure.resourcemanager.loganalytics.models.PublicNetworkAccessType;
+import com.azure.resourcemanager.loganalytics.models.WorkspaceCapping;
+import com.azure.resourcemanager.loganalytics.models.WorkspaceFeatures;
+import com.azure.resourcemanager.loganalytics.models.WorkspaceSku;
+import com.azure.resourcemanager.loganalytics.models.WorkspaceSkuNameEnum;
+import com.azure.resourcemanager.network.NetworkManager;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.workloads.models.ManagedRGConfiguration;
+import com.azure.resourcemanager.workloads.models.Monitor;
+import com.azure.resourcemanager.workloads.models.RoutingPreference;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class WorkloadsManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region RESOURCE_REGION = Region.US_EAST2;
+    private static final Region MANAGED_REGION = Region.US_EAST;
+    private static final Region APPLICATION_REGION = Region.US_WEST;
+    private static final String MANAGED_RG_NAME = "managerRG" + randomPadding();
+    private String resourceGroupName = "rg" + randomPadding();
+    private LogAnalyticsManager logAnalyticsManager;
+    private NetworkManager networkManager;
+    private WorkloadsManager workloadsManager;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        logAnalyticsManager = LogAnalyticsManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        networkManager = NetworkManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        workloadsManager = WorkloadsManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        resourceManager.resourceGroups().define(MANAGED_RG_NAME).withRegion(MANAGED_REGION).create();
```

Comment:
could you re-try, without this line?

Reply:
ResourceGroupNotFound exception occrus when without this line, the detail infomation check [here](https://github.com/Azure/azure-sdk-for-java/files/12010990/workloads-live-test.log)

Reply:
because you are deleting it here 
```
        resourceManager.resourceGroups().beginDeleteByName(MANAGED_RG_NAME);
```

Reply:
The `MANAGED_RG_NAME` related code has been removed in the new version.